.github/workflows/draft_release.yml

```diff
@@ -0,0 +1,19 @@
+name: Draft Release
+
+on:
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  draft-release:
```

Comment:
Not sure why this entire file shows as a change... It was from a previous PR... I did rename this job name from deploy-staging to draft-release, but that's all.